package.json

```diff
@@ -152,6 +152,7 @@
     "webdriverio": "^8.32.4",
     "wrangler": "^3.51.2"
   },
+  "packageManager": "pnpm@8.15.8",
```

Comment:
what's this for?

Reply:
If this is set, you don't have to declare it for the usage of `pnpm/action-setup`.